1-org/envs/shared/README.md

```diff
@@ -37,6 +37,20 @@ You can choose not to enable the Data Access logs by setting variable `data_acce
 1. Run `terraform plan` and review output.
 1. Run `terraform apply`
 
+### Run tf-wrapper.sh script locally
```

Comment:
I believe that `Run tf-wrapper.sh script locally` should actually be `Run terraform locally` and replace the original section.
@bharathkkb @rjerrems

Reply:
Yup agreed. We can have a small oneliner on why we have `tf-wrapper.sh` if needed below this.